app/uk/gov/hmrc/transitmovementsauditing/controllers/AuditController.scala

```diff
@@ -16,19 +16,25 @@
 
 package uk.gov.hmrc.transitmovementsauditing.controllers
 
+import akka.stream.Materializer
+import akka.stream.scaladsl.Source
+import akka.util.ByteString
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 import play.api.mvc.Action
-import play.api.mvc.AnyContent
 import play.api.mvc.ControllerComponents
+import uk.gov.hmrc.transitmovementsauditing.controllers.stream.StreamingParsers
+
 import javax.inject.Inject
 import javax.inject.Singleton
 import scala.concurrent.Future
 
 @Singleton()
-class MicroserviceHelloWorldController @Inject() (cc: ControllerComponents) extends BackendController(cc) {
+class AuditController @Inject() (cc: ControllerComponents, implicit val materializer: Materializer) extends BackendController(cc) with StreamingParsers {
```

Comment:
The `implicit` paramaters should be in their own parameter list.

Reply:
Oops, of course